src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,13 +1,33 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Truck
- * Do not remove no-field constructor
- */
 public class Truck extends Machine {
+    private double bodyVolume;
+    private boolean isTrailerCovered;
+
     public Truck() {
     }
 
+    public Truck(boolean isTrailerCovered, double bodyVolume) {
+        this.isTrailerCovered = isTrailerCovered;
+        this.bodyVolume = bodyVolume;
+    }
+
+    public double getBodyVolume() {
+        return bodyVolume;
+    }
+
+    public void setBodyVolume(double bodyVolume) {
+        this.bodyVolume = bodyVolume;
+    }
+
+    public boolean isTrailerCovered() {
+        return isTrailerCovered;
+    }
+
+    public void setTrailerCovered(boolean trailerCovered) {
+        isTrailerCovered = trailerCovered;
+    }
```

Comment:
If you don`t use getters or setters, you can clear it

Reply:
done